packages/core/strapi/lib/services/entity-service/index.js

```diff
@@ -217,7 +217,7 @@ const createDefaultImplementation = ({ strapi, db, eventHub, entityValidator })
     const componentsToDelete = await getComponents(uid, entityToDelete);
 
     await db.query(uid).delete({ where: { id: entityToDelete.id } });
-    await deleteComponents(uid, { ...entityToDelete, ...componentsToDelete });
+    await deleteComponents(uid, componentsToDelete);
```

Comment:
In entityToDelete the id was not there?

Reply:
yes! But it wasn't for the bulk delete.

I though it was going to complicate things doing this in the bulkDelete, when you have multiple entities. 
So my proposal is to return the entity id in `getComponents` instead

Reply:
After discussing it with you I made the changes!
It didn't make sense to return the entity ID in `getComponents`. Now we specify if we want to loadComponentss when calling `deleteComponents` 👍